2200024/4406_모음.py

```diff
@@ -0,0 +1,12 @@
+T = int(input())
+
+for t in range(1, T+1) :
+    s = input()
+
+    for _ in range(5) :
+        s = s.replace('a',"")
+        s = s.replace('e',"")
+        s = s.replace('i',"")
+        s = s.replace('o',"")
+        s = s.replace('u',"")
```

Comment:
```suggestion
    s = s.replace('a',"")
    s = s.replace('e',"")
    s = s.replace('i',"")
    s = s.replace('o',"")
    s = s.replace('u',"")
```
`for`문은 작성하지 않으셔도 괜찮습니다  😄